annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastProcessor.java

```diff
@@ -32,8 +44,21 @@ public synchronized void init(ProcessingEnvironment processingEnv) {
 
     @Override
     public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
+        if (roundEnv.processingOver()) {
+            try {
+                addServiceEntry(SERVICE_TYPENAME, SERVICE_PROVIDER,
+                    nameClassMap.values().stream().toArray(Element[]::new));
+            } catch (IOException e) {
+                processingEnv.getMessager().printMessage(Diagnostic.Kind.ERROR,
+                    String.format("Unable to create or update java services file for %s service", SERVICE_TYPENAME));
+            }
+        }
         if (annotations.contains(fastTypeElement)) {
-            HashMap<String, Element> nameClassMap = collectFastTypes(roundEnv);
+            if (isGeneratedCreatorImpl) {
+                processingEnv.getMessager().printMessage(Diagnostic.Kind.ERROR,
+                    "CreatorImpl is already generated. new types aren't allowed");
```

Comment:
@FastType annotation processor can not handle classes with @FastType annotation added after first processing round.  It may be caused by activity of other annotation processors.